### ISSUE TITLE
Hardware acceleration in v9 reducing game framrates

### ISSUE THREAD

**Issue by TheEngineerGuy** [JosefNemec/Playnite#2709]
### Bug Description

Hardware acceleration when enabled in v9, consistently and measurably reduces frame in games by 3-4% in GPU limited games. This is quite easily reproducible.

v8 Hardware acceleration does not seem to reduce the frame rates, it is only noticeable in v9.

Furthermore, disabling hardware acceleration in v9, produces same frame rates as not using playnite to launch the game at all. Which brings me to an understanding that hardware acceleration keeps certain GPU resources tied up when games are running.

### To Reproduce

Enable HW acceleration, run a heavy GPU limited game.
Disable HW acceleration, run a heavy GPU limited game.

You will notice 3-4% frame rate drop with HW acceleration.

### Diagnostics ID

e248c9d6-4fae-49fe-9c37-7136013c755a

**Comment by JosefNemec** [comment]
There was no change in V9 that could cause this. If you are using custom theme, try switching to the default one to see if the issue remains. This is most likely caused by unoptimized theme.

**Comment by TheEngineerGuy** [comment]
That is a good point. I thought about it, but did not test that scenario as HW acceleration disable with same theme did not impact the frames. 

Taking your suggestion, I will try this without the custom theme.

**Comment by JosefNemec** [comment]
I can't reproduce it. Playnite reports exactly 0% GPU usage when in idle. What benchmark are you using?

**Comment by TheEngineerGuy** [comment]
AC Valhalla is the one I am using. It is quite consistent. 

You are right, gpu usage as you say shows as zero, but frame rate certainly drops when in games with gpu limited scenarios, if he acceleration is enabled.

**Comment by JosefNemec** [comment]
I don't own Valhalla, but I tried it with Odyssey's benchmark and I can't measure any difference when Playnite is running or not running.

**Comment by TheEngineerGuy** [comment]
Yes, Odyssey benchmark itself is quite inconsistent. The runt o run variance of that benchmark is upwards of 15% depending if clouds are in the scene or not. Plus odyssey is DX11, and Valhalla is DX12, so a significant difference in API calls.

Let me check a few other games as well.

**Comment by TheEngineerGuy** [comment]
![PlayniteGUIGPUUsage](https://user-images.githubusercontent.com/16248146/147802398-85c686fc-6048-46d0-ad14-27136b0e2c8a.png)


A quick view of GPU usage with HW Acceleration enabled.

**Comment by JosefNemec** [comment]
Are you using some custom theme? The only thing I can think of is that you are using some custom theme that's not optimized and it is running something (maybe animations) even when not focused or when a game is running.

I just can't reproduce any framerate drops when Playnite is running and when no 3rd party add-ons are in play.

**Comment by JosefNemec** [comment]
Just to add to that, I tried multiple games. My only guess is that it's most likely some compatibility issues with some other software you are using (if you are sure it's not caused by 3rd party extension or theme). Maybe something that injects overlays into apps (Playnite is DX9 app so it could be incorrectly detected as a game), like MSI Afterburner (or other RivaTuner forks), that's known to cause issues in the past.

The other thing is that you are saying it started happening with P9, but there was no change in P9 that affects GUI performance at all. The only performance related change was to how we store database files on the disk and there's no way that could introduce GPU related issues.

**Comment by JosefNemec** [comment]
Closing per my previous messages and also fact that we are completely switching UI in Playnite 11.
